### ISSUE TITLE
[ini export] the learning rate scheduler has a problem of being exported

### ISSUE THREAD

**Issue by DonghakPark** [nnstreamer/nntrainer#2296]
Currently, reading with ini format works normally, **but the results are different when exporting.**
In particular, the learning rate scheduler has a problem of being exported in a completely different way than intended.
Below is the difference when the mnist.ini file is imported and exported to the mnist_model.ini file again.

original
```.ini
[Optimizer]
Type = adam
beta1 = 0.9                         # beta 1 for adam
beta2 = 0.999                       # beta 2 for adam
epsilon = 1e-7                      # epsilon for adam

[LearningRateScheduler]
type=step
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
```

exported
```.ini
[optimizer]
beta1 = 0.9
beta2 = 0.999
epsilon = 1e-07
iteration = 4,6,15
learning_rate = 0.000100,0.000090,0.000070,0.000050
torch_ref = false
type = adam
```
